superset/connectors/sqla/models.py

```diff
@@ -1302,32 +1304,41 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
                 )
                 inner_groupby_exprs = []
                 inner_select_exprs = []
-                for gby_name, gby_obj in groupby_exprs_sans_timestamp.items():
+                for gby_name, gby_obj in groupby_series_columns.items():
+                    if series_columns and gby_name not in series_columns:
+                        continue
                     inner = self.make_sqla_column_compatible(gby_obj, gby_name + "__")
                     inner_groupby_exprs.append(inner)
                     inner_select_exprs.append(inner)
 
                 inner_select_exprs += [inner_main_metric_expr]
                 subq = select(inner_select_exprs).select_from(tbl)
-                inner_time_filter = dttm_col.get_time_filter(
-                    inner_from_dttm or from_dttm,
-                    inner_to_dttm or to_dttm,
-                    time_range_endpoints,
-                )
-                subq = subq.where(and_(*(where_clause_and + [inner_time_filter])))
+                inner_time_filter = []
+
+                if dttm_col and not db_engine_spec.time_groupby_inline:
```

Comment:
I have no idea what `db_engine_spec.time_groupby_inline` does, but I can only assume it relates to temporal filters in subqueries. If this turns out not to be the case I can improve the documentation in a subsequent PR.